draft-thomson-webpush-protocol.xml

```diff
@@ -328,7 +338,40 @@ Cache-Control: max-age:864000, private
 ]]></artwork>
       </figure>
     </section>
+    <section anchor="receipt_subscription" title="Subscribing to Receipts">
+      <t>
+        An application server sends a POST request to its configured push
```

Comment:
That's one possibility and a variation on the original proposal to have the PS return a receipt resource to the UA for distribution to the AS. Can you explain why it cannot be configured in this fashion? I expect that there are more AS-PS configuration requirements lurking.

To clarify, you're proposing a new link relation for receiptsubscriptions that is distributed by the UA to its AS?